frontend/src/modules/SimulationTimeSeriesMatrix/settings.tsx

```diff
@@ -64,6 +66,9 @@ export function Settings({ moduleContext, workbenchSession }: ModuleFCProps<Stat
     const setParameterIdent = moduleContext.useSetStoreValue("parameterIdent");
     const setVectorSpecifications = moduleContext.useSetStoreValue("vectorSpecifications");
 
+    // Transitions
+    const [isPending, startTransition] = React.useTransition();
```

Comment:
Rename to `[isPendingGetParameters, startGetParametersTransistion]` or something similar? To specify what transistion the hook is intendent/utilized for.